### PR TITLE
Emulate github power up

### PR DESCRIPTION
**Pull request by Surzhko**
- Left comment on github issue to trello card
- Attach github issue to trello card

Only one difference, that comment on github without trello icon

### CODE REVIEW

File: lib/github_workflow/cli.rb [hunk 1]
```diff
@@ -207,6 +207,9 @@ def create_issue_from_trello_card
         response = JSON.parse(github_client.post("repos/#{user_and_repo}/issues", issue_params.to_json).body)
 
         @issue_id = response["number"]
+
+        github_client.post("/repos/#{user_and_repo}/issues/#{@issue_id}/comments?access_token=#{oauth_token}", { body: trello_card.short_url }.to_json)
```
Comment (Surzhko) [inline lib/github_workflow/cli.rb:211]: Will update `access_token` now

**Comment by rpmilevskiy** [comment]
:+1: Awesome!

File: lib/github_workflow/cli.rb [hunk 2]
```diff
@@ -251,7 +254,7 @@ def set_trello_card(type:)
             Trello::Board.find(project_config["trello_board_id"])
           end
 
-        @trello_card = trello_board.cards.detect { |card| card.short_id == options["card_number"].to_i }
+        @trello_card = trello_board.find_card(options["card_number"].to_i)
```
Comment (BrianBorge) [inline lib/github_workflow/cli.rb:257]: 👍

**Review by BrianBorge — COMMENTED** [review]
Looks good.  My only comment is, do you think it would be better to include the link to the Trello Issue as part of the body of the GH Issue instead of a comment?

**Review by benliscio — APPROVED** [review]
🚀

File: lib/github_workflow/cli.rb (hunk 2 above)
Comment (Surzhko) [reply lib/github_workflow/cli.rb:257]: Currently it's emulate behavior of Github Power Up, which uses comment, but Yes, we could add link as a part of the body

Comment (BrianBorge) [reply lib/github_workflow/cli.rb:257]: Got it.  Ben explained it to me as well.

**Pull request merged**